api/DOMConfiguration.json

```diff
@@ -1,244 +0,0 @@
-{
-  "api": {
-    "DOMConfiguration": {
-      "__compat": {
-        "mdn_url": "https://developer.mozilla.org/docs/Web/API/DOMConfiguration",
```

Comment:
This went together with `document.domConfig`, which was once in Firefox:
https://github.com/mdn/browser-compat-data/blob/c1f99a1ee3166e343cd509d7e205a06f890fdc46/api/Document.json#L3525-L3532

But it was so long ago now that the entry can still be removed. Can you roll that into this change a well?